src/common/test-util/src/lib.rs

```diff
@@ -20,6 +20,7 @@ use std::sync::LazyLock;
 
 pub mod ports;
 pub mod temp_dir;
+pub mod wal;
```

Comment:
How about place these utils close the `log-store`?

Reply:
Some utils are used both by log-store and common-meta. So it cannot be placed into log-store or there will be a cyclic dependency.

Reply:
Wal utils are now removed from `test-util` crate since most of them are deleted.